website/docs/usage/examples.md

```diff
@@ -32,6 +32,14 @@ Calculate and set the next semantic version (i.e. _bump the version_) for the un
 git cliff --bump
 ```
 
+Simply calculate and return next semantic version:
+
+```bash
+# Semver: {MAJOR}.{MINOR}.{PATCH}
+# "fix:" increments PATCH, "feat:" increments MINOR and "scope!" (breaking changes) increments MAJOR
+git cliff --bumped-version
+```
```

Comment:
We can merge this with this example above to avoid the duplicate explanation of SemVer.